worlds/banjo_tooie/Options.py

```diff
@@ -0,0 +1,513 @@
+
+class OpenHag1(DefaultOnToggle):
+    """HAG 1 boss fight is opened when Cauldron Keep is opened. Only 55 jiggies are needed to win."""
+    display_name = "HAG 1 Open"
+
+class RandomizeWorlds(Toggle):
+    """Worlds will open in a randomized order. Randomized Moves and Puzzle Skip Required."""
+    display_name = "Randomize Worlds"
+
+class RandomizeWorldZones(Toggle):
+    """World Entrances will warp you to a different world. This does not affect Chuffy."""
+    display_name = "Randomize World Entrances"
+
+class RandomizeStopnSwap(Toggle):
+    """Mystery Eggs, their rewards, and the Ice Key are scattered across the MultiWorld."""
+    display_name = "Randomize Stop n Swap"
+
+class SkipToT(Choice):
+    """Choose whether to play the full quiz, start at round 3, or skip it."""
+    display_name = "Tower of Tragedy Quiz"
+    option_full = 0
+    option_skip = 1
+    option_round_3 = 2
+    default = 1
+
+class LogicType(Choice):
+    """Choose your logic difficulty and difficulty of tricks you are expected to perform to reach certain areas.
+    Please be aware that if you plan on randomizing worlds with BK Moves in the pool, you cannot use "intended" logic.""" 
+    display_name = "Logic Type"
+    option_intended = 0
+    option_easy_tricks = 1
+    option_hard_tricks = 2
+    option_glitches = 3
+    default = 0
+
+class Silos(Choice):
+    """Choose if you want IoH Silos to be closed, randomly open 1 or enable all. If you enabled Randomized Worlds with BK Moves randomized and
+       silos set to none, it will be enforced to one."""
+    display_name = "Open Silos"
+    option_none = 0
+    option_one = 1
+    option_all = 2
+    default = 0
+
+class VictoryCondition(Choice):
+    """Choose which victory condition you want
+    HAG1: Unlock the HAG1 fight and defeat Gruntilda
+    Minigame Hunt: Clear the 14 minigames and the final Canary Mary race in Cloud Cuckcoo Land to collect Mumbo Tokens
+    Boss Hunt: Kill the 8 world bosses and collect their Mumbo Tokens
+    Jinjo Family Rescue: Rescue Jinjo Families to collect their prized Mumbo Tokens
+    Wonderwing Challenge: Collect all 32 Mumbo Tokens across all boss fights, mini games and every Jinjo family
+        to gain access to HAG1 and Defeat Grunty. The Ultimate Banjo Tooie experience!!
+    Token Hunt: Mumbo's Tokens are scattered around the world. Help him find them"""
+    display_name = "Victory Condition"
+    option_hag1 = 0
+    option_minigame_hunt = 1
+    option_boss_hunt = 2
+    option_jinjo_family_rescue = 3
+    option_wonder_wing_challenge = 4
+    option_token_hunt = 5
+    default = 0
+
+class MinigameHuntLength(Range):
+    """How many Mumbo Tokens are needed to clear the Minigame Hunt.
+    Choose a value between 1 and 15"""
+    display_name = "Minigame Hunt Length"
+    range_start = 1
+    range_end = 15
+    default = 14
+
+class BossHuntLength(Range):
+    """How many Mumbo Tokens are needed to clear the Boss Hunt.
+    Choose a value between 1 and 8"""
```

Comment:
```suggestion
    Choose a value between 1 and 8."""
```
I'm not entirely sure it's necessary to state the range here, but I'd need to look at another yaml to confirm that.